less/type.less

```diff
@@ -238,6 +238,67 @@ ol {
   }
 }
 
+ul.tw-list-styled {
+  padding: 0;
+  margin: 32px 0;
+  list-style-type: none;
+  li {
+    position: relative;
+    padding-left: 40px;
+    margin-bottom: 16px;
+    counter-increment: step-counter;
+    strong {
+      color: #2ED06E;
+    }
+    &::before {
+      font-family: "TransferWise-IconFont";
+      position: absolute;
+      left: 0;
+      width: 24px;
+      content: "\e916";
```

Comment:
I think there are some designs that use a cross rather than tick, we should use a modifier class , something like `.circle-list-check`, and `.circle-list-close` (the icon names used icomoon set) to choose which icon we use.  Mike claims they are never mixed in the same list so this can be on the parent ul.  The close/cross variant is also red, not green.